action/protocol/poll/protocol.go

```diff
@@ -396,6 +422,23 @@ func (p *governanceChainCommitteeProtocol) getGravityHeight(height uint64) (uint
 	return p.electionCommittee.HeightByTime(blkTime)
 }
 
+func (p *governanceChainCommitteeProtocol) mergeDelegates(cand state.CandidateList, name [][]byte, votes []*types.Vote) error {
```

Comment:
You may want to make votes to be a map instead? Then the following merging logic could be simpler

Reply:
this is fixed in latest change